update-center-source.properties

```diff
@@ -5,11 +5,17 @@ scanners=scannercli,scannermsbuild,scannerant,scannermaven,scannergradle,scanner
 devVersion=10.8-SNAPSHOT
 publicVersions=9.9,9.9.1,9.9.2,9.9.3,9.9.4,9.9.5,9.9.6,9.9.7,10.0,10.1,10.2,10.2.1,10.3,10.4,10.4.1,10.5,10.5.1,10.6,10.7
 privateVersions=4.2,4.3,4.3.1,4.3.2,4.3.3,4.4,4.4.1,4.5,4.5.1,4.5.2,4.5.4,4.5.5,4.5.6,4.5.7,5.0,5.0.1,5.1,5.1.1,5.1.2,5.2,5.3,5.4,5.5,5.6,5.6.1,5.6.2,5.6.3,5.6.4,5.6.5,5.6.6,5.6.7,6.0,6.1,6.2,6.3,6.3.1,6.4,6.5,6.6,6.7,6.7.1,6.7.2,6.7.3,6.7.4,6.7.5,6.7.6,6.7.7,7.0,7.1,7.2,7.2.1,7.3,7.4,7.5,7.6,7.7,7.8,7.9,7.9.1,7.9.2,7.9.3,7.9.4,7.9.5,7.9.6,8.0,8.1,8.2,8.3,8.3.1,8.4,8.4.1,8.4.2,8.5,8.5.1,8.6,8.6.1,8.7,8.7.1,8.8,8.9,8.9.1,8.9.2,8.9.3,8.9.4,8.9.5,8.9.6,8.9.7,8.9.8,8.9.9,8.9.10,9.0,9.0.1,9.1,9.2,9.2.1,9.2.2,9.2.3,9.2.4,9.3,9.4,9.5,9.6,9.6.1,9.7,9.7.1,9.8,
+sqcb=24.12.0.100206
 
 ltsVersion=9.9.7
 ltaVersion=9.9.7
 pastLtaVersion=8.9.10
 
+24.12.0.100206.description=Multi-Quality Rule and Standard modes, support for C\\#13/.Net 9, Java 22, Python 3.13, Kotlin 2.0, and many new rules for test and main code
```

Comment:
@lukasz-jarocki-sonarsource I tried to escape it with `\\` but I don't know how to test

Reply:
Let me know if you want to learn later how to test it on your local computer. 
Just write # as it is, without escaping.

Reply:
Do we know how it will render in the UI? Looking at the previous descriptions, I'm not aware of a problem with #.